.github/workflows/code-quality.yml

```diff
@@ -146,38 +77,19 @@ jobs:
         with:
           path: ./node_modules
           key: "${{ runner.os }}-node_modules-${{ hashFiles('package-lock.json') }}-${{ hashFiles('**/schema.prisma') }}"
-      - name: Reset nx
-        run: npx nx reset
-      - name: Run build
-        run: npm run build
-
-  cypress:
-    runs-on: ubuntu-latest
-    if: ${{ false }}
-    needs:
-      - test
-      - lint
-      - dependency-review
-    strategy:
-      # https://github.com/cypress-io/github-action/issues/48
-      fail-fast: false
-      matrix:
-        # Use 2 parallel instances
-        containers: [1, 2]
-    steps:
-      - name: Checkout repository
-        uses: actions/checkout@v4
       - name: Start services
         env:
           DB_USER: ${{ env.DB_USERNAME }}
           DB_PASSWORD: ${{ env.DB_PASSWORD }}
         run: |
           cd development
+          chmod +x ./init/elasticsearch/init.sh
+          sed -i 's/- \.\/volumes\/elasticsearch\/data:\/usr\/share\/elasticsearch\/data//g' ./docker-compose.yaml
           docker compose up -d db oidc elasticsearch
-      - name: Setup
-        run: |
-          npm run prisma -- reset -f
-          # npm run start:server &> /dev/null &
+          sleep 120
```

Comment:
This necessary?

Reply:
I have reduced but is still needed for services to be up and running before Cypres starts.